picard/util/tags.py

```diff
@@ -165,653 +343,1119 @@ def names(self, selector=None):
         shortdesc=N_('Encoded By'),
+        longdesc=N_('The person or organization that encoded the track.'),
+        is_from_mb=False,
     ),
     TagVar(
         'encodersettings',
         shortdesc=N_('Encoder Settings'),
+        longdesc=N_('The settings used when encoding the track.'),
+        is_from_mb=False,
     ),
     TagVar(
         'engineer',
         shortdesc=N_('Engineer'),
+        longdesc=N_('The names of the engineers associated with the track.'),
     ),
     TagVar(
         'extension',
-        shortdesc=N_('File extension'),
+        shortdesc=N_('File Extension'),
+        longdesc=N_("The file's extension."),
         is_hidden=True,
         is_preserved=True,
         is_tag=False,
+        is_from_mb=False,
     ),
     TagVar(
         'file_created_timestamp',
-        shortdesc=N_('File created timestamp'),
+        shortdesc=N_('File Created Timestamp'),
+        longdesc=N_('The file creation timestamp in the form `YYYY-MM-DD HH:MM:SS` as reported by the file system.'),
         is_hidden=True,
         is_preserved=True,
         is_tag=False,
+        is_from_mb=False,
     ),
     TagVar(
         'file_modified_timestamp',
-        shortdesc=N_('File modified timestamp'),
+        shortdesc=N_('File Modified Timestamp'),
+        longdesc=N_('The file modification timestamp in the form `YYYY-MM-DD HH:MM:SS` as reported by the file system.'),
         is_hidden=True,
         is_preserved=True,
         is_tag=False,
+        is_from_mb=False,
     ),
     TagVar(
         'filename',
-        shortdesc=N_('File name'),
+        shortdesc=N_('File Name'),
+        longdesc=N_('The name of the file without extension.'),
         is_hidden=True,
         is_preserved=True,
         is_tag=False,
+        is_from_mb=False,
     ),
     TagVar(
         'filepath',
         shortdesc=N_('File Path'),
+        longdesc=N_('Full path and name of the file.'),
         is_hidden=True,
         is_tag=False,
+        is_from_mb=False,
     ),
     TagVar(
         'filesize',
-        shortdesc=N_('File size'),
+        shortdesc=N_('File Size'),
+        longdesc=N_('Size of the file in bytes.'),
         is_file_info=True,
         is_hidden=True,
         is_tag=False,
+        is_from_mb=False,
     ),
     TagVar(
         'format',
-        shortdesc=N_('File format'),
+        shortdesc=N_('File Format'),
+        longdesc=N_('Media format of the file (e.g.: MPEG-1 Audio).'),
         is_file_info=True,
         is_hidden=True,
         is_preserved=True,
         is_tag=False,
+        is_from_mb=False,
     ),
     TagVar(
         'gapless',
         shortdesc=N_('Gapless Playback'),
+        longdesc=N_('Indicates whether or not there are gaps between the recordings on the release.'),
+        is_from_mb=False,
     ),
     TagVar(
         'genre',
         shortdesc=N_('Genre'),
+        longdesc=N_(
+            'A multi-value tag containing the specified genre information from MusicBrainz.'
```

Comment:
I wonder, while we are at it, if it would be convenient to have a `TagVar.is_multi_value` flag.

Reply:
I think that's a good idea.  I'll add it in.